### PR TITLE
clarify tiup playground in quick-start-with-tidb.md

### PR DESCRIPTION
**Pull request by yilongrong**
多个用户问过，tiup playground  的数据保留问题。建议可以加上注意事项来说明。

### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [x] v3.1 (TiDB 3.1 versions)
- [x] v3.0 (TiDB 3.0 versions)
- [x] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: quick-start-with-tidb.md [hunk 1]
```diff
@@ -107,6 +107,7 @@ title: TiDB 数据库快速上手指南
 > **注意：**
 >
 > TiUP Playground 默认监听 `127.0.0.1`，服务仅本地可访问；若需要使服务可被外部访问，你可以通过 `--host` 参数指定监听网卡绑定外部可访问的 IP。
+> TiUP Playground 关闭后不会保留数据，每次使用相当于新的集群。
```
Comment (lucklove) [inline quick-start-with-tidb.md:110]:
```suggestion
> 默认情况下，TiUP Playground 关闭后不会保留数据，每次使用相当于新的集群。这是因为 TiUP 在组件运行结束之后会清理产生的数据，如果想要不清理数据以便下次继续使用，可使用 `tiup --tag <tag> playground ...`，详情请参考 [TiUP Reference](/tiup/tiup-reference.md)
```

**Comment by TomShawn** [comment]
This PR is duplicated with https://github.com/pingcap/docs-cn/pull/6049.

**Pull request closed without merging**